DedicatedServer/Scripts/Modes/Trackmania/FlagRush.Script.txt

```diff
@@ -1358,6 +1363,11 @@ Void HandleCommands() {
 			}
 			case CSmModeEvent::EType::OnPlayerRemoved: {
 				FlagRush_UI::UpdateRadar();
+
+				if (Players.count == 0 && Spectators.count == 0) {
+					// Everyone left but we are in warmup, just reset scores
+					TeamPointsProgression::EndMatch();
```

Comment:
Should also reset players scores etc. I think the match progression could need some work again, i.e. marking rounds, maps and matches valid.